.circleci/config.yml

```diff
@@ -5,18 +5,118 @@ references:
     client pricehistory bot nlp servers broker eventstore nginx
 
 commands:
+
+  # rewriting internal 'checkout' command to solve the issue with triggering forked PRs build with API
```

Comment:
this is mostly a copy-paste from original `checkout` command that CircleCi had (the difference is in handling branches for forks, see below)